hbase-shell/src/main/ruby/shell/commands/count.rb

```diff
@@ -60,17 +71,27 @@ def count(table, params = {})
         # If the second parameter is an integer, then it is the old command syntax
         params = { 'INTERVAL' => params } if params.is_a?(Integer)
 
+        # Try to be nice and convert a string to a bool
+        if params.include?('CACHE_BLOCKS') and params['CACHE_BLOCKS'].is_a?(String)
+          if params['CACHE_BLOCKS'].downcase == 'true'
+            params['CACHE_BLOCKS'] = true
+          elsif params['CACHE_BLOCKS'].downcase == 'false'
+            params['CACHE_BLOCKS'] = false
+          end
```

Comment:
Maybe worth throwing an error back if it isn't true or false?

Reply:
Good idea.